.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "weekly"
+    allowed_updates:
+      - match:
+          dependency_type: "production"
```

Comment:
so this excludes devDependencies?

Reply:
it seems to be like that based on documentation. I wanted to update the `develop` branch and then trigger the Dependabot to check.